src/contracts/data-bus/data-bus.client.ts

```diff
@@ -0,0 +1,147 @@
+import { Contract, providers, Signer, utils } from 'ethers';
+import { EventDataMap, eventMappers } from './data-bus.serializer';
+import { MessagesDataMap, MessagesNames } from './data-bus.serializer';
+import * as eventsAbi from '../../abi/data-bus.abi.json';
+import { TransactionResponse } from '@ethersproject/abstract-provider';
+import { DATA_BUS_REQUEST_TIMEOUT } from './data-bus.constants';
+
+export class DataBusClient {
```

Comment:
Is the order of messages being sent important? What if they are reordered?

Reply:
Order of messages is not important